src/main/scala/com/hungry/cars/domain/ValidatedCreateUserRequest.scala

```diff
@@ -0,0 +1,10 @@
+package com.hungry.cars.domain
+
+case class ValidatedCreateUserRequest(
```

Comment:
Nie widze sensu istnienia takiej case class'y. Moge bez problemu utworzyc obiekt takiej klasy ktory jest niepoprawny. Tak naprawde niczym sie to nie rozni od CreateUserRequest na poziomie typow.